ParisBNF/abb/BNFabb4.xml

```diff
@@ -0,0 +1,349 @@
+                    </physDesc>
+                        
+                        <history>
+                            <origin>                                
+                                <ref target="#p1">Unit 1</ref> can be dated palaeographically to the
+                                <origDate notBefore="1650" notAfter="1750">Second half of the 17th/first half of the 18th century, 
+                                possibly contemporary to <ref target="#p2">Unit 2</ref></origDate>.
+                                The four leaves that make out <ref target="#p1">Unit 1</ref> are probably remains 
+                                of a manuscript which did 
+                                not survive but was similar to  <ref target="#p2">Unit 2</ref> in terms of size and layout. 
+                                At an unknown time, <ref target="#p1">Unit</ref> was introduced into the binding of <ref target="#p2">Unit 2</ref>.
+                            </origin>
+                        </history>
+                        </msPart>               
+                                                               
+                                                              
+                                <msPart xml:id="p1_i2">
+                                    <msIdentifier/>
+                                    <msContents>
+                                        <summary/>
+                                        
+                                        <msItem xml:id="p2_i1">          
+                            <locus from="5ra" to="146vb"/>
+                            <title type="complete" ref="LIT1146Argano"/> 
+                            <textLang mainLang="gez"/>                            
+                        </msItem>                     
+                                                                        
+                                <msItem xml:id="p2_i2">
+                            <locus from="147ra" to="154rb"/>
+                            <title type="incomplete" ref="LIT3990Prayer"/>
+                            <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rub">እሴብሕ፡ እግዚእየ፡</hi> ኢየሱስ፡ ክርስቶስ፡ ምስ<hi rend="rub">ለ፡ 
+                                አቡከ፡ ኄር፡ ወመንፈስ፡</hi> ቅዱስ፨ 
+                                በስብሐት፡ ማር፡ ፈለክሲኖስ፡ ዘሀገረ፡ ምንቡዢር፨ ዘይጸልዩ፡ ባቲ፡ ኵሉ፡ ሰብእ፡ በእንተ፡ መድኃኒተ፡ ነፍሶሙ፨ 
+                                <hi rend="rub">ስብሐት፡</hi> ለከ፡ እግዚአብሔር፡ አብ፨ <hi rend="rub">ስብሐት፡</hi>
+                                ለከ፡ እግዚአብሔር፡ ወልድ፨<hi rend="rub">ስብሐት፡</hi> ለከ፡ እግዚአብሔር፡ መንፈስ፡ ቅዱስ፨ ስብሐት፡ ለከ፡ ኦሥሉስ፡ 
+                                በግጻዌ፡ ወ፩፡ በጽምረት፡ ወዕሩይ፡ በመለኮት፨</incipit>
+                        </msItem> 
+                        
+                        <msItem xml:id="p2_i3">
+                            <locus from="154rb" to="159vb"/>
+                            <title type="incomplete" ref="LIT4274AkkonuB"/> 
```

Comment:
can you add some note as to the meaning of "incomplete"?

Reply:
Corrected, it is actually complete.